capstone/src/main/resources/static/map.js

```diff
@@ -0,0 +1,56 @@
+/* Function to create Google Map in map.html page */
+async function createMap() {
+  var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
+  const campusMap = new google.maps.Map(
+    document.getElementById('map'),
+    {center: princetonLatLng, zoom: 16});
+
+   google.maps.event.addListener(campusMap, 'click', function(event) {
+    placeMarkerAndPan(event.latLng, campusMap);
+   });
+
+   const response = await fetch('get-all-events');
+   const jsonEvents = await response.json();
+   jsonEvents.forEach(event => createMarker(event, campusMap));
+
+  }
+
+/* Function to create Mini Map in event.html page */
+  function createEventPlacementMap() {
+    var princetonLatLng = {lat: 40.3428452, lng: -74.6568153};
+    const campusMap = new google.maps.Map(
+      document.getElementById('eventMap'),
+      {center: princetonLatLng, zoom: 16});
+
+     google.maps.event.addListener(campusMap, 'click', function(event) {
+      placeMarkerAndPan(event.latLng, campusMap);
```

Comment:
I guess you're still working on populating the lat and long in the event form based on the click, which would go right after this line.

Reply:
resolved!